binchihua/src/orch/orch.c

```diff
@@ -0,0 +1,37 @@
+#include "../../../libchihua-msg/include/orchestrator.h"
+#include "../ini/ini.h"
+#include "opt.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+/* A function that takes care of printing the lines of the ini file  */
+// NOLINTNEXTLINE
+static int ini_handler_func(void *user, const char *section, const char *name, const char *value) {
+        printf("[%s] %s = %s\n", section, name, value);
+        return EXIT_FAILURE;
+}
+
+int main(int argc, char *argv[]) {
+        fprintf(stdout, "Hello from orchestrator!\n");
+
+        uint16_t accept_port = 0;
+        char *configPath = NULL;
```

Comment:
Variables and functions should be in ```snake_case```.

Reply:
Same as [here](https://github.com/engelmi/orch/pull/50#pullrequestreview-1236111764). This is a pretty small change, though. I'll change it and check other places as well.